docker/install/ubuntu_install_python_package.sh

```diff
@@ -32,10 +32,12 @@ pip3 install \
     packaging \
     Pillow \
     pytest \
+    tlcpack-sphinx-addon==0.2.1 \
     pytest-profiling \
     pytest-xdist \
     requests \
     scipy \
     synr==0.6.0 \
     six \
+    ninja \
```

Comment:
Can we `apt-get install ninja-build` rather than pip?

Reply:
See https://github.com/apache/tvm/pull/10246#discussion_r807238655, is there any advantage to sticking with `apt`?

Reply:
`apt` has better security and stability vs `pip`, especially given we install all these dependencies blindly - I'd generally favour  using the OSes package manager until we actually need to work around it.

Reply:
i'm okay with sticking with `apt` for OS packages. for Python packages, i prefer we move to `pip` once we have properly recorded sha256 for those packages we want to install.

Reply:
the pip `ninja` is just a thin wrapper to distribute the binary so in this case `apt` makes sense

Reply:
going to resolve this one as i think we've addressed it